### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by Olli781** [tiagohm/nebulosa#328] [General]
I find the project very exciting and would really like to install/test it. Unfortunately, my programming skills are not good. So my question is: is there some kind of installation guide? Thanks, Oliver

**Comment by tiagohm** [comment]
You can download and install the generated artifact https://github.com/tiagohm/nebulosa/actions/runs/7661689131 while a release is not available.

**Comment by tiagohm** [comment]
Now, you can download here: https://github.com/tiagohm/nebulosa/releases/latest
